artman/cli/main.py

```diff
@@ -227,6 +227,14 @@ def parse_args(*args):
         action='store_true',
         help='[Optional] When specified, artman will skip the remote '
         'publishing step.', )
+    parser_publish.add_argument(
+        '--local-repo-dir',
+        type=str,
+        default=None,
+        help='[Optional] When specified, artman will skip the lookup local '
```

Comment:
s/skip the lookup local publishing/neither look up the publishing/
s/, nor/nor/

Reply:
done